### DISCUSSION TITLE
Noticing dangerous site warning when using a redirect

### DISCUSSION THREAD

**Discussion by anshul-kai** [vercel/next.js#59965] [Help] [answered]
### Summary

My `next.config.js` file sets up a short URI as a redirect to another longer route. The longer route opens just fine but the short URI makes Chrome believe that it's a phishing link. Anybody else notice something similar?

<img width="690" alt="Screenshot 2023-12-27 at 6 06 59 AM" src="https://github.com/vercel/next.js/assets/16167317/5a6a0c92-7bd5-4145-940d-559487aa1507">

### Additional information

```js
const nextConfig = {
  redirects: async () => {
    return [
      {
        source: '/shortRoute',
        destination: '/longRoute/longPageName',
        permanent: true
      }
    ];
  }
};
```

**Comment by nicholasgriffintn** [comment, marked as answer]
This means google has flagged your site for some reason, if you go in google search console there should be a "security issues" page that might have more details.

Outside of that, you can only remove the cause or submit a request for a change to google: https://safebrowsing.google.com/safebrowsing/report_error/

**Reply by aman-techie** [reply]
Hi,
It has happened with me as well.
Did chrome remove this error for your short url ?
If yes, How much time did they take ?

**Reply by anshul-kai** [reply]
Yes, they removed my warning within hours. Thanks @nicholasgriffintn for the feedback.

**Reply by WilliamVivs** [reply]
running same issue, asked for google to remove the advice
